.gitignore

```diff
@@ -1,3 +1,5 @@
-/tests/.DS_Store
-.DS_Store
+# build
 /build
+
+# macOS
```

Comment:
Is there a reason for this change?

Reply:
Just to make clear to exclude all `.DS_Store` files (which is `macOS` specific).

Reply:
Sure, but they were already excluded, weren't they?

Reply:
Yes, but why then make an extra statement for `/tests/.DS_Store`?

Reply:
Historical reasons, I suppose.

(BTW, this is a good example where a helpful commit message would clear things up quickly: "update gitignore" --> "Removed redundant paths in gitignore.")